content/academy/apify_platform/actors.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Actors
+description: What is an actor? How do we create them? Learn the basics of what actors are, how they work, and try out an actor yourself right on the Apify platform!
+menuWeight: 5.3
+paths:
+- apify-platform/actors
+---
+
+# [](#actors) Actors
+
+After you've followed the **Getting started** and **Apify CLI** lessons, and familiarized yourself with the Apify SDK in [**Web scraping for beginners**]({{@link web_scraping_for_beginners/crawling/pro_scraping.md}}), you're almost ready to start creating actors! But before we get into that, let's discuss what an actor is, and a bit about how they work.
+
+## [](#what-is-an-actor) What's an actor?
+
+When you deploy your script to the Apify platform, it is then called an **actor**, which is simply a serverless microservice that accepts an input and produces an output. Actors can run for a few seconds, hours or even infinitely. An actor can perform anything from a simple action such as filling out a web form or sending an email, to complex operations such as crawling an entire website and removing duplicates from a large dataset.
+
+Once an actor has been pushed to the Apify platform, they can be shared to the world through the [Apify Store](https://apify.com/store), and even monetized after going public.
+
+> Though the majority of actors that are currently on the Apify platform are scrapers, crawlers, or automation software, actors are not limited to just scraping. They are just pieces of code running in Docker containers, which means they can be used for nearly anything.
+
+## [](#actors-on-platform) Actors on the Apify platform
+
+For a super quick and dirty understanding of what a publicized actor looks like, and how it works, let's run an SEO audit of **apify.com** using the [SEO audit actor](https://apify.com/drobnikj/seo-audit-tool).
```

Comment:
IMO "published" would be the better term here, instea of "publicized"

Reply:
True